draft-ietf-httpbis-digest-headers.md

```diff
@@ -633,29 +633,49 @@ Content-Type: application/json
 ~~~
 {: title="Request containing a JSON object without any content coding"}
 
+Compression is an efficient way to reduce
+the content length.
+
 ~~~ http-message
 PUT /entries/1234 HTTP/1.1
 Host: foo.example
 Content-Type: application/json
 Content-Encoding: gzip
 
-H4sIAItWyFwC/6tWSlSyUlAypANQqgUAREcqfG0AAAA=
+0x1F0x8B0x080x000xA50xB40xBD0x620x020xFF\
+0xAB0x560x4A0x540xB20x520x500x320xA40x03\
+0x500xAA0x050x000x440x470x2A0x7C0x6D0x00\
+0x000x00
 ~~~
-{: title="Request containing a gzip-encoded JSON object"}
+{: title="Request containing a gzip-encoded JSON object" #ex-put-gz}
 
-Now the same content conveys a malformed JSON object, because the request does
-not indicate a content coding.
+Removing the content coding from the same request
+causes the content to become malformed.
+The server replies with an error.
```

Comment:
I guess we can argue it's not the "same request", because the message is different :smile:, so how about something like

```suggestion
Sending the compressed form of the content 
without the correct content coding means that
the content is malformed.
In this case, the server can reply with an error.
```